spec/system/admin/products_v3/update_spec.rb

```diff
@@ -58,9 +58,6 @@
       end
 
       # Unit popout
-      fill_in "Unit value", with: ""
```

Comment:
As we don't have required validation as per [this](https://github.com/openfoodfoundation/openfoodnetwork/pull/12710/files#r1703198190), so we can remove its expectation

Reply:
Can the case of blank unit value now be added here?

https://github.com/openfoodfoundation/openfoodnetwork/blob/master/spec/system/admin/products_v3/update_spec.rb#L251